css/css-scroll-snap/support/scroll-target-padding-001-iframe.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<title>iframe for #target and scroll-snap-padding with snapping off (y</title>
+<style type='text/css'>
+  html, body {
+    margin: 0; padding: 0;
+  }
+  html {
+    /* to make failing more obvious */
+    background: 0 1em / 100% 1em linear-gradient(red, red) repeat-x fixed;
+    /* avoid anti-aliasing issues */
+    font: 20px/1 sans-serif;
+  }
+  div {
+    height: 1em;
+  }
+  html    { scroll-padding: 2em 0 1em; } /* snap area is exact fit for snapport  */
```

Comment:
Shouldn't this be `scroll-padding: 1em 0 2em;` here?

Reply:
No, the green stripe is not the target element. It is above the target element.
I'm trying to remember why I did it that way, I think there was a reason...